streamer/src/streamer.rs

```diff
@@ -171,6 +279,7 @@ pub fn responder(
                     last_print = now;
                     errors = 0;
                 }
+                stats.maybe_submit(name);
```

Comment:
There is a datapoint already using the `name`. I'm not sure if it conflicts, seems like it should be ok though.

Reply:
It's using `name` to log `"errors"` in the `responder` loop. There shouldn't be a conflict.